pom.xml

```diff
@@ -28,6 +26,13 @@
     <!-- JVM params for test execution -->
     <argLine>-Duser.language=en</argLine>
 
+    <!-- Spring Boot / Cloud / Stream deps -->
```

Comment:
I thought that we wanted every dependency version managed in the root pom. Has this changed?